test/coverage/datamodel/UUriSerializerTest.cpp

```diff
@@ -12,9 +12,9 @@
 #include <gtest/gtest.h>
 #include <up-cpp/datamodel/serializer/UUri.h>
 
-namespace {
+namespace uprotocol::datamodel::serializer::uuri {
```

Comment:
Leave the tests in the anonymous namespace
```suggestion
namespace {
```

Reply:
Done